.github/workflows/tests.yaml

```diff
@@ -97,7 +102,7 @@ jobs:
         if: github.event_name == 'schedule'
         run: |
           echo "::set-env name=TEST_COUNT::100"
-          echo "::set-env name=TEST_ARGS::-timeout 240m"
+          echo "::set-env name=TEST_ARGS::-timeout 240m -json"
```

Comment:
Why add `-json` two times: here and above at `echo "::set-env name=TEST_ARGS::-json"` line?